pyproject.toml

```diff
@@ -211,9 +211,11 @@ reportUnnecessaryTypeIgnoreComment = true
 reportMissingTypeStubs = false
 exclude = [
     "docs/**/*.py",
+    "examples/**/*.py",
```

Comment:
This is necessary to ignore the code in the examples repo and avoid a bunch of type checking we don't care about